packages/core/App.tsx

```diff
@@ -56,6 +57,24 @@ export default function App(props: AppProps) {
         HTMLDivElement
     >();
 
+    React.useEffect(() => {
+        const beforeUnloadHandler = (event: BeforeUnloadEvent) => {
+            // Modern browser alert: Does not allow custom messages
+            event.preventDefault();
+
+            // Legacy support (e.g. Chrome/Edge < 119): Allows custom messages
+            // Deprecated, does not affect modern browsers
+            event.returnValue =
+                "Edits to external data sources will not be saved. Make sure to download the result in order to keep your changes";
```

Comment:
Not attached to this wording, essentially just a placeholder. Open to suggestions

Reply:
Could we add a 'save' button to the alert?  I suppose it might be premature.  What happens if the user makes a series of changes?  Will the alert pop up every time?

Reply:
Combined Sean's rephrasing with Lyndsay's suggestions in [5e1a136](https://github.com/AllenInstitute/biofile-finder/pull/550/commits/5e1a136d1c5039f1fac9a1ecc7a302ab0c214330)